modules/oauth/src/main/java/org/mule/extension/oauth2/internal/tokenmanager/TokenManagerConfig.java

```diff
@@ -23,15 +28,35 @@
  *
  * It can be referenced to access the state inside a flow for custom processing of oauth dance content.
  */
+@Alias("token-manager-config")
+@XmlHints(allowTopLevelDefinition = true)
 public class TokenManagerConfig implements Initialisable, MuleContextAware {
 
   public static AtomicInteger defaultTokenManagerConfigIndex = new AtomicInteger(0);
+
+  /**
+   * Identifier for the token manager configuration.
+   */
+  @ConfigName
   private String name;
+
+  /**
+   * References an object store to use for storing oauth context data
+   */
+  @Parameter
+  @Optional
+  @Alias("objectStore-ref")
   private ListableObjectStore objectStore;
```

Comment:
so there would be no way to provide a custom object store for this object to use?